File: Programacion/Python/Ej_Recursividad/Ej6.py
def fibonacci(m):
    if m <= 0:
        return 0
    elif m == 1:
        return 1
    else:
        return fibonacci(m-1) + fibonacci(m-2)

def suma_fibonacci(n):
    if n <= 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fibonacci(n) + suma_fibonacci(n-1)

n1 = int(input("Fibonacci dime un número: "))
print("Tu fibonacci: ", suma_fibonacci(n1))

File: DAM_1/DAM_2º-año_1º/Programación/UD4-Ej1/src/Ej6.java
import java.util.Scanner;

/**
 * Crear una función que tenga como parámetros de entrada 2 números enteros
 * y como parámetro de salida un doble con la media aritmética de ambos.
 * **/
public class Ej6 {
    static double mediaAritmetica(int n1, int n2){
        return (n1+n2)/2.0;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Dime un número: ");
        int num1 = scanner.nextInt();

        System.out.print("Dime otro número: ");
        int num2 = scanner.nextInt();

        double media = mediaAritmetica(num1,num2);

        System.out.println("La media es: " + media);

    }
}
